dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -110,6 +114,7 @@ def __init__(self, options=None):
             "histogram_and_quantiles":
                 NumericStatsMixin._get_histogram_and_quantiles,
             "num_zeros": NumericStatsMixin._get_num_zeros,
+            "mode": NumericStatsMixin._get_mode,
```

Comment:
instead of calculating at profile time, I think we can do this at report time (since so simple). we won't need to time it then.